build.sbt

```diff
@@ -0,0 +1,22 @@
+name := "http4s-demo"
+
+version := "1.0-SNAPSHOT"
+
+scalaVersion := "2.11.7"
+
+val rhoVersion = "0.12.0"
```

Comment:
Should we be using the "a" versions of this and http4s to be on scalaz-7.2?  I guess our naming scheme implies the scalaz-7.1 version as a "default," but that's unfortunate.